### PR TITLE
Add Chamorro language support

### PR DESCRIPTION
**Pull request by amandabeiner**
Adds support for the Chamorro language for the Guam build

#### Linked issues:

[GAEN-251](https://pathcheck.atlassian.net/jira/software/projects/GAEN/boards/33?selectedIssue=GAEN-251)

#### Screenshots:

Simulator Screen Shot - iPhone SE (2nd generation) - 2020-09-08 at 14 22 45
Simulator Screen Shot - iPhone SE (2nd generation) - 2020-09-08 at 14 23 17

### CODE REVIEW

**Review by johnschoeman — APPROVED** [review]
LGTM

File: src/configuration/copy.json [hunk 1]
```diff
@@ -1,11 +1,14 @@
 {
   "welcome_message": {
-    "en": ""
```
Comment (aledustet) [inline src/configuration/copy.json:3]: probably don't want to include this one here.

File: src/locales/languages.ts [hunk 2]
```diff
@@ -9,6 +9,7 @@ import { StorageUtils } from "../utils"
 
 import ar from "./ar.json"
 import da from "./da.json"
+import ch from "./ch.json"
```
Comment (aledustet) [inline src/locales/languages.ts:12]: I think we need to include the `ko` file for the Korean translation here

**Review by aledustet — APPROVED** [review]
One question about the `ko` translation otherwise looks gewd

**Comment by johnschoeman** [comment]
looks like this work got duplicated here: https://github.com/Path-Check/gaen-mobile/pull/350, closing this one as the other seems to have the most up to date translations as well.

**Pull request closed without merging**